### PR TITLE
adding docstrigs

### PR DESCRIPTION
**Pull request by matthiasto**
adding auto doc to
AlphaBeta
AIMEC

### CODE REVIEW

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 1]
```diff
@@ -101,8 +112,26 @@ def getFileList(path2Folder):
 
 
 def mainAIMEC(cfgPath, cfg):
-    """
-    Main logic for AIMEC postprocessing
+    """ Main logic for AIMEC postprocessing
+
+    Reads the requiered files location for ana3AIMEC postpocessing
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:117]:
```suggestion
    Reads the required files location for ana3AIMEC postpocessing
```

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 2]
```diff
@@ -406,18 +484,38 @@ def makeTransfoMat(rasterTransfo):
 
 
 def getSArea(rasterTransfo):
-    """
+    """ Get the s curvilinear coorsinate and Area on the new raster
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:487]:
```suggestion
    """ Get the s curvilinear coordinate and area on the new raster
```

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 3]
```diff
@@ -537,8 +656,30 @@ def assignData(fnames, rasterTransfo, interpMethod):
 # -----------------------------------------------------------
 
 def analyzeData(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
-    """
-    Analyse pressure and depth deskewed data
+    """ Analyse pressure and depth transformed data
+
+    Main function for result analysis
+    Plots the desiered figures and saves results as txt files
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:662]:
```suggestion
    Plots the desired figures and saves results as txt files
```

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 4]
```diff
@@ -537,8 +656,30 @@ def assignData(fnames, rasterTransfo, interpMethod):
...
+        dictionary containing pressure, velocity and flow depth rasters after
+        transformation
+    cfgPath: dict
+        path to data to analyse
+    cfgFlags: configparser
+        configparser with plot and write flags
+
+    Returns
+    -------
+    Plots the desiered figures and saves results as txt files
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:680]:
```suggestion
    Plots the desired figures and saves results as txt files
```

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 5]
```diff
@@ -713,20 +884,38 @@ def analyzeFields(rasterTransfo, pLim, newRasters, cfgPath):
...
-            -FN: ref = False sim2 = True
-            -FP: ref = True sim2 = False
-            -TN: ref = False sim2 = False
+
+    Parameters
+    ----------
+    rasterTransfo: dict
+        transformation information
+    resAnalysis: dict
+        resAnalysis dictionnary containing all results to update
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:896]:
```suggestion
        resAnalysis dictionary containing all results to update
```

File: avaframe/com2AB/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+"""AlphaBeta module kernel code"""
```
Comment (fso42) [inline avaframe/com2AB/__init__.py:1]:
```suggestion
"""AlphaBeta kernel"""
```

File: avaframe/com2AB/com2AB.py [hunk 7]
```diff
@@ -1,7 +1,5 @@
 """
     Main logic for Alpha beta computational module
```
Comment (fso42) [inline avaframe/com2AB/com2AB.py:2]:
```suggestion
    Main module for Alpha beta

```

**Review by fso42 — APPROVED** [review]
Just spelling stuff...

**Comment by matthiasto** [comment]
> Just spelling stuff...

Thanks, spelling is not my strength...

**Pull request merged**